pkg/store/prometheus.go

```diff
@@ -496,9 +499,39 @@ func (p *PrometheusStore) LabelValues(ctx context.Context, r *storepb.LabelValue
 		return &storepb.LabelValuesResponse{Values: []string{l}}, nil
 	}
 
-	vals, err := p.client.LabelValuesInGRPC(ctx, p.base, r.Label, nil, r.Start, r.End)
-	if err != nil {
-		return nil, err
+	var (
+		sers []map[string]string
+		err  error
+	)
+
+	vals := []string{}
+	version := p.promVersion()
```

Comment:
`p.promVersion` might return some garbage value if the `m.BuildVersion` doesn't run correctly, so let's make sure `version` is a correct version string, and not some garbage value.